lightning-background-processor/src/lib.rs

```diff
@@ -371,8 +372,7 @@ macro_rules! define_run_body {
 
 					have_pruned = true;
 				}
-				let prune_timer = if have_pruned { NETWORK_PRUNE_TIMER } else { FIRST_NETWORK_PRUNE_TIMER };
-				last_prune_call = $get_timer(prune_timer);
+				last_prune_call = $get_timer(NETWORK_PRUNE_TIMER);
 			}
 
 			if $timer_elapsed(&mut last_scorer_persist_call, SCORER_PERSIST_TIMER) {
```

Comment:
Wondering is it still necessary to keep this regular 30s persist if we're persisting on every update to the scorer?

Reply:
The timer for the scorer was set to persist every 30s, so I just changed it to an hour to be similar with the graph.